02-basic-programming/README.md

```diff
@@ -28,21 +28,25 @@ Com este curso, você desenvolverá dois programas que têm aplicação em nosso
 1. **Cifra de César**: você criará um programa que criptografa e decodifica mensagens de acordo com o [algoritmo da Cifra de César](https://pt.wikipedia.org/wiki/Cifra_de_C%C3%A9sar) \(_Caesar cipher_\).
 2. **Validação de cartão de crédito**: você fará um programa capaz de verificar se um cartão de crédito é válido de acordo [com o algoritmo de Luhn](https://es.wikipedia.org/wiki/Algoritmo_de_Luhn).
 
-## Syllabus
+## Programa de Estudos
```

Comment:
We should use the keyboard `Syllabus` as it is what the parser looks for in any locale.